src/useApi.js

```diff
@@ -0,0 +1,36 @@
+export default function useApi() {
+  const baseurl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  const scoreurl = 'games/superawesomegame/scores/';
+  const outputdiv = document.querySelector('.scores');
+  const userinp = document.querySelector('.userinp');
+  const scoreinp = document.querySelector('.scoreinp');
+  function loaddata() {
+    axios.get(baseurl + scoreurl).then((response) => {// eslint-disable-line
+      outputdiv.innerHTML = '';
+      response.data.result.forEach((item) => {
+        outputdiv.innerHTML
+          += `<li>name:${item.user} score:${item.score}</li>`;
+      });
+    });
+  }
+
+  function adduser() {
```

Comment:
- [ ] Kindly, make sure to use arrow functions instead of the function keyword.

Reply:
Updated to use arraw functions.